### ISSUE TITLE
Mouse Cell Atlas reads are aligned to mm9 whereas basenji predicts from mm10

### ISSUE THREAD

**Issue by GrossTor** [calico/basenji#124]
The [Cross-species regulatory sequence activity prediction project](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5932613/) uses [bigwig tracks from the mouse cell atlas](https://atlas.gs.washington.edu/mouse-atac/data/#ucsc-trackhub-and-bigwigs). Those were generated from ATACseq reads that were aligned to the mm9 genome assembly. However, basenji tries to predict those tracks from the sequence of the mm10 genome assembly (other mouse tracks e.g. from Fantom5 are aligned to mm10 as well).

Could you explain how this discrepancy was resolved? I see multiple solutions:  

- use [Crossmap](http://crossmap.sourceforge.net/) to map the bigwig files to mm10
- retrieve the fastq files from the Mouse Cell Atlas and generate mm10 aligned bigwigs with the basenji data pipeline
- align all other mouse tracks to mm9 and only work on mm9

What would you recommend?

**Comment by davek44** [comment]
That's correct. Crossmap seems good. I converted to bedgraph, then used UCSC liftover.

**Comment by GrossTor** [comment]
Makes sense. Thank you!
